portal_partner_select_all/wizard/portal_wizard.py

```diff
@@ -14,12 +14,9 @@ class PortalWizard(models.TransientModel):
     @api.onchange('set_all_users')
     def onchange_set_all_users(self):
         """Toggle between select all partners and the default"""
-        if not self.set_all_users:
-            for user in self.user_ids:
-                user.in_portal = (
-                    user.partner_id.user_ids and
-                    self.portal_id in user.partner_id.user_ids[0].groups_id
-                )
-        else:
+        if self.set_all_users:
             not_in_portal = self.user_ids.filtered(lambda x: not x.in_portal)
             not_in_portal.update({'in_portal': True})
+        else:
+            self.user_ids = None
```

Comment:
Is it necessary to set it to None before the next statement?

Reply:
it is necrssary: self._default_user_ids() is implemented with `0,0` not `6,0`
With each toogle user_ids would be duplicated

Reply:
[Odoo 12 Implementation _default_user_ids](https://github.com/odoo/odoo/blob/8c7d0266cc6d7bd0569721e92128096e64d9c361/addons/portal/wizard/portal_wizard.py#L46)

Reply:
I'd say the assignment is brand new for each call: https://github.com/odoo/odoo/blob/8c7d0266cc6d7bd0569721e92128096e64d9c361/addons/portal/wizard/portal_wizard.py#L36

Reply:
Thank you, changes made using has_group